impl/random/rtthread.h

```diff
@@ -0,0 +1,38 @@
+// Please enable RT_USING_HWCRYPTO and RT_HWCRYPTO_USING_RNG
```

Comment:
What does that mean? Is it possible to check for that in the scons file and abort/explain how to enable it if they aren't enabled?

Reply:
> What does that mean? Is it possible to check for that in the scons file and abort/explain how to enable it if they aren't enabled?

Actually these two options are automatically enabled once the pacakge libhygrogen is selected.

Thus, this it just a friently reminder that libhydrogen depends on hardware PRNG, or this line can be removed.

Reply:
This line hase be removed, because these options will automatically be enabled by the package manager.